spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -324,6 +329,8 @@ def aggregate_per_slice_or_level(metric, mask=None, slices=[], levels=[], distan
             agg_metric[slicegroup]['VertLevel'] = None
         else:
             agg_metric[slicegroup]['VertLevel'] = vertgroups[slicegroups.index(slicegroup)]
+            if agg_metric[slicegroup]['VertLevel'][0] is None:
+                agg_metric[slicegroup]['VertLevel'] = None
```

Comment:
> I'm trying to understand commit https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4004/commits/0c21b168b275d01caa83b152d0b7018b7445c58e. It seems like it's made up of three parts?

3. **?**

    My apologies, but on first read, I don't quite understand this change. What's the purpose of this change?

Reply:
Here, if there was no level corresponding to the slice, the tuple gives : `(None,)` which means that is the .csv file, it was written `None` instead of leaving it empty. This is just to put None instead of (None,)

Reply:
Ahhh, okay. So basically:

- `(None,)` --> "None" cell
- `None` --> Empty cell

And we prefer an empty cell here?

Reply:
yes, to be consistent with the other cells in the .csv file. If there is no segmentation for a particular slice, the cell is left empty when no metric are computed